src/popup/pages/MultisigDetails.vue

```diff
@@ -102,15 +103,18 @@ export default defineComponent({
       nonce, signers, confirmationsRequired,
     } = activeMultisigAccount.value || {} as IMultisigAccount;
 
+    const consensus = computed(() => (
+      `${confirmedBy.length}/${confirmationsRequired} ${root.$t('of')} ${signers.length}`
+    ));
```

Comment:
I think that code could be extracted to the composable as it is used also on dashboard and will be used on transaction list